### ISSUE TITLE
Test failures for "Run examples" integration jobs

### ISSUE THREAD

**Issue by lbianchi-lbl** [IDAES/idaes-pse#278]
This happened in several PRs (e.g. #208, #274, #277). I'm looking into this now; we can use this issue to keep track of the progress.

## What we know about the error

- Seems to be coming from `Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb`
- `Kernel died` errors do not leave a stack trace in the build log like e.g. a normal runtime exception would, making it trickier to figure out the underlying cause
```log
 2021-04-01 19:01:57,904 DEBUG - [Worker 0 Main-Loop]: Run worker function: begin
2021-04-01 19:01:57,904 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb: begin
2021-04-01 19:01:57,905 INFO - [Worker 0] Running: DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb
2021-04-01 19:01:57,908 DEBUG - [Worker 0] Found tag 'remove_cell' in cell 23
2021-04-01 19:01:57,908 DEBUG - [Worker 0] notebook 'DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb' has test cell(s)
2021-04-01 19:01:57,908 DEBUG - [Worker 0] run notebook in temporary directory: /tmp/tmplywhqham
2021-04-01 19:01:57,914 DEBUG - [Worker 0] removing tag(s) <remove_cell, exercise'> from notebook: DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb
2021-04-01 19:01:58,024 DEBUG - [Worker 0] writing stripped notebook: DMF_2_flash_unit_Model_with_NRTL_solution
2021-04-01 19:01:58,030 INFO - [Worker 0] Stripped tags from: DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb
2021-04-01 19:01:58,036 DEBUG - [Worker 0] checking if cached: /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution.html src=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb
2021-04-01 19:01:58,036 INFO - [Worker 0] Running notebook: DMF_2_flash_unit_Model_with_NRTL_solution.ipynb
2021-04-01 19:01:58,036 DEBUG - [Worker 0] parsing '/tmp/tmplywhqham/DMF_2_flash_unit_Model_with_NRTL_solution.ipynb'
2021-04-01 19:01:58,036 DEBUG - [Worker 0] executing '/tmp/tmplywhqham/DMF_2_flash_unit_Model_with_NRTL_solution.ipynb'
2021-04-01 19:02:02,059 ERROR - [Worker 0] Failed due to error: Kernel died
2021-04-01 19:02:02,067 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb: end, ok=False duration=4.2s
2021-04-01 19:02:02,072 DEBUG - [Worker 0 Main-Loop]: Run worker function: end
```
### UPDATE

- After a closer look, it seems that the two `DMF_...` notebooks are executed out of order, i.e. `DMF_2_...` is run before `DMF_1_...`. The dependency between these two notebooks is a known cause of errors, and this shouldn't happen
```log
2021-04-01 19:13:01,580 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb: begin
2021-04-01 19:13:01,580 INFO - [Worker 0] Converting: DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb
2021-04-01 19:13:01,692 INFO - [Worker 0] Stripped tags from: DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb
2021-04-01 19:13:01,704 INFO - [Worker 0] Running notebook: DMF_2_flash_unit_Model_with_NRTL_solution.ipynb
2021-04-01 19:13:05,719 ERROR - [Worker 0] Failed due to error: Kernel died
2021-04-01 19:13:05,719 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb: end, ok=False duration=4.2s
2021-04-01 19:13:05,719 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb: begin
2021-04-01 19:13:05,720 INFO - [Worker 0] Converting: parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb
2021-04-01 19:13:05,885 INFO - [Worker 0] Stripped tags from: parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb
2021-04-01 19:13:05,885 INFO - [Worker 0] Running notebook: parameter_estimation_NRTL_using_unit_model_solution.ipynb
2021-04-01 19:14:43,882 INFO - [Worker 0] Exporting notebook 'parameter_estimation_NRTL_using_unit_model_solution.ipynb' to directory /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst
2021-04-01 19:14:44,295 INFO - [Worker 0] generate Sphinx doc wrapper for parameter_estimation_NRTL_using_unit_model_solution => /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_unit_model_solution_doc.rst
2021-04-01 19:14:44,575 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb: end, ok=True duration=98.9s
2021-04-01 19:14:44,576 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb: begin
2021-04-01 19:14:44,576 INFO - [Worker 0] Converting: DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb
2021-04-01 19:14:44,759 INFO - [Worker 0] Stripped tags from: DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb
2021-04-01 19:14:44,765 INFO - [Worker 0] Running notebook: DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution.ipynb
2021-04-01 19:15:06,575 INFO - [Worker 0] Exporting notebook 'DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution.ipynb' to directory /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst
2021-04-01 19:15:07,016 INFO - [Worker 0] generate Sphinx doc wrapper for DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution => /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst/DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_doc.rst
2021-04-01 19:15:07,397 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb: end, ok=True duration=22.8s
2021-04-01 19:15:07,403 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_state_block_solution_testing.ipynb: begin
2021-04-01 19:15:07,408 INFO - [Worker 0] Converting: parameter_estimation_NRTL_using_state_block_solution_testing.ipynb
2021-04-01 19:15:07,560 INFO - [Worker 0] Stripped tags from: parameter_estimation_NRTL_using_state_block_solution_testing.ipynb
2021-04-01 19:15:07,572 INFO - [Worker 0] Running notebook: parameter_estimation_NRTL_using_state_block_solution.ipynb
2021-04-01 19:15:48,601 INFO - [Worker 0] Exporting notebook 'parameter_estimation_NRTL_using_state_block_solution.ipynb' to directory /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst
2021-04-01 19:15:48,961 INFO - [Worker 0] generate Sphinx doc wrapper for parameter_estimation_NRTL_using_state_block_solution => /home/runner/work/examples-pse/examples-pse/docs_test/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_state_block_solution_doc.rst
2021-04-01 19:15:49,851 INFO - [Worker 0] Convert notebook name=/home/runner/work/examples-pse/examples-pse/src/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_state_block_solution_testing.ipynb: end, ok=True duration=42.5s
```
### UPDATE 2

This seems to be confirmed by comparing the collection order (i.e. without actually running the notebooks) from the test runs on Ubuntu and Windows:

#### Ubuntu

```log
Run pytest --verbose -m unit tests/
============================= test session starts ==============================
platform linux -- Python 3.8.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /opt/hostedtoolcache/Python/3.8.8/x64/bin/python
cachedir: .pytest_cache
rootdir: /home/runner/work/examples-pse/examples-pse, configfile: pytest.ini
collecting ... collected 37 items / 2 deselected / 35 selected

tests/test_basic.py::test_import_syntax PASSED                           [  2%]
tests/test_build_script.py::test_notify PASSED                           [  5%]
tests/test_build_script.py::test_script_usage PASSED                     [  8%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Advanced/CustomProperties/air_separation_case_study.ipynb] PASSED [ 11%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Advanced/DataRecon/econ_recon_testing.ipynb] PASSED [ 14%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Advanced/DataRecon/econ_parmest_testing.ipynb] PASSED [ 17%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Advanced/DataRecon/boiler_flowsheet_recon_testing.ipynb] PASSED [ 20%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Advanced/CustomUnitModels/custom_compressor_testing.ipynb] PASSED [ 22%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Advanced/CustomUnitModels/custom_heater_testing.ipynb] PASSED [ 25%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/SurrMod/ALAMO/ALAMO_six_hump_camel.ipynb] PASSED [ 28%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/SurrMod/RIPE/clc_nb.ipynb] PASSED [ 31%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/SurrMod/RIPE/RIPE_isothermal_CSTR.ipynb] PASSED [ 34%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/SurrMod/PySMO/PySMO_basics.ipynb] PASSED [ 37%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Pecos/data_quality_control_testing.ipynb] PASSED [ 40%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/UnitModels/heater_testing.ipynb] PASSED [ 42%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/UnitModels/compressor_testing.ipynb] PASSED [ 45%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/UnitModels/turbine_testing.ipynb] PASSED [ 48%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/UnitModels/mixer_testing.ipynb] PASSED [ 51%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/UnitModels/pump_testing.ipynb] PASSED [ 54%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/Tools/data_management_framework.ipynb] PASSED [ 57%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/MatOpt/bimetallic_nanocluster_design.ipynb] PASSED [ 60%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/MatOpt/bifunctional_surface_design.ipynb] PASSED [ 62%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/MatOpt/surface_design.ipynb] PASSED [ 65%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/MatOpt/monometallic_nanocluster_design.ipynb] PASSED [ 68%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/MatOpt/nanowire_design.ipynb] PASSED [ 71%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Examples/MatOpt/metal_oxide_bulk_design.ipynb] PASSED [ 74%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Advanced/ParamEst/DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb] PASSED [ 77%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb] PASSED [ 80%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Advanced/ParamEst/DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb] PASSED [ 82%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Advanced/ParamEst/parameter_estimation_NRTL_using_state_block_solution_testing.ipynb] PASSED [ 85%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Basics/HDA_flowsheet_solution_testing.ipynb] PASSED [ 88%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Basics/introduction_short_solution_testing.ipynb] PASSED [ 91%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Basics/introduction_solution_testing.ipynb] PASSED [ 94%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/Tutorials/Basics/flash_unit_solution_testing.ipynb] PASSED [ 97%]
tests/test_notebooks.py::test_parse_notebook[/home/runner/work/examples-pse/examples-pse/tests/../src/notebook_index.ipynb] PASSED [100%]
```

#### Windows

```log
Run pytest --verbose -m unit tests/
============================= test session starts =============================
platform win32 -- Python 3.8.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- c:\hostedtoolcache\windows\python\3.8.8\x64\python.exe
cachedir: .pytest_cache
rootdir: D:\a\examples-pse\examples-pse, configfile: pytest.ini
collecting ... collected 37 items / 2 deselected / 35 selected

tests/test_basic.py::test_import_syntax PASSED                           [  2%]
tests/test_build_script.py::test_notify PASSED                           [  5%]
tests/test_build_script.py::test_script_usage PASSED                     [  8%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Advanced\\CustomProperties\\air_separation_case_study.ipynb] PASSED [ 11%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Advanced\\CustomUnitModels\\custom_compressor_testing.ipynb] PASSED [ 14%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Advanced\\CustomUnitModels\\custom_heater_testing.ipynb] PASSED [ 17%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Advanced\\DataRecon\\boiler_flowsheet_recon_testing.ipynb] PASSED [ 20%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Advanced\\DataRecon\\econ_parmest_testing.ipynb] PASSED [ 22%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Advanced\\DataRecon\\econ_recon_testing.ipynb] PASSED [ 25%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\MatOpt\\bifunctional_surface_design.ipynb] PASSED [ 28%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\MatOpt\\bimetallic_nanocluster_design.ipynb] PASSED [ 31%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\MatOpt\\metal_oxide_bulk_design.ipynb] PASSED [ 34%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\MatOpt\\monometallic_nanocluster_design.ipynb] PASSED [ 37%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\MatOpt\\nanowire_design.ipynb] PASSED [ 40%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\MatOpt\\surface_design.ipynb] PASSED [ 42%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Pecos\\data_quality_control_testing.ipynb] PASSED [ 45%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\SurrMod\\ALAMO\\ALAMO_six_hump_camel.ipynb] PASSED [ 48%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\SurrMod\\PySMO\\PySMO_basics.ipynb] PASSED [ 51%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\SurrMod\\RIPE\\clc_nb.ipynb] PASSED [ 54%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\SurrMod\\RIPE\\RIPE_isothermal_CSTR.ipynb] PASSED [ 57%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\Tools\\data_management_framework.ipynb] PASSED [ 60%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\UnitModels\\compressor_testing.ipynb] PASSED [ 62%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\UnitModels\\heater_testing.ipynb] PASSED [ 65%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\UnitModels\\mixer_testing.ipynb] PASSED [ 68%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\UnitModels\\pump_testing.ipynb] PASSED [ 71%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Examples\\UnitModels\\turbine_testing.ipynb] PASSED [ 74%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\notebook_index.ipynb] PASSED [ 77%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Advanced\\ParamEst\\DMF_1_for_parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb] PASSED [ 80%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Advanced\\ParamEst\\DMF_2_flash_unit_Model_with_NRTL_solution_testing.ipynb] PASSED [ 82%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Advanced\\ParamEst\\parameter_estimation_NRTL_using_state_block_solution_testing.ipynb] PASSED [ 85%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Advanced\\ParamEst\\parameter_estimation_NRTL_using_unit_model_solution_testing.ipynb] PASSED [ 88%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Basics\\flash_unit_solution_testing.ipynb] PASSED [ 91%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Basics\\HDA_flowsheet_solution_testing.ipynb] PASSED [ 94%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Basics\\introduction_short_solution_testing.ipynb] PASSED [ 97%]
tests/test_notebooks.py::test_parse_notebook[D:\\a\\examples-pse\\examples-pse\\tests\\..\\src\\Tutorials\\Basics\\introduction_solution_testing.ipynb] PASSED [100%]
```
From these, it looks like paths are sorted by default on Windows, but not on Linux. I'm trying again after adding a call to `sorted` where the notebooks are collect, and see if this fixes the issue

**Comment by lbianchi-lbl** [comment]
Update: the notebook seems to run fine when running it manually on my system (Ubuntu 20.04). I'll try other ways to reproduce the error.

**Comment by lbianchi-lbl** [comment]
@eslickj I'm trying to run the notebook in a local Docker container (`python:3.7`, based on Debian Buster), but I'm getting ipopt failures. Is this method OK to test whether ipopt is available within IDAES?

```sh
$ python -c 'from pyomo.environ import SolverFactory as Sf; print(Sf("ipopt").available())'
WARNING: Could not locate the 'ipopt' executable, which is required for solver
    ipopt
False
```

**Comment by lbianchi-lbl** [comment]
This has been fixed by IDAES/examples-pse#40.
